src/LatteTemplateResolver/NetteApplicationUIPresenter.php

```diff
@@ -74,14 +50,16 @@ private function findTemplateFilePath(ReflectionClass $reflectionClass, string $
     {
         $shortClassName = $reflectionClass->getShortName();
         $presenterName = str_replace('Presenter', '', $shortClassName);
-        $dir = $this->findTemplateDir($reflectionClass);
+        $dir = $this->getClassDir($reflectionClass);
         if ($dir === null) {
             return null;
         }
 
         $templateFileCandidates = [
             $dir . '/templates/' . $presenterName . '/' . $actionName . '.latte',
             $dir . '/templates/' . $presenterName . '.' . $actionName . '.latte',
+            $dir . '/../templates/' . $presenterName . '/' . $actionName . '.latte',
```

Comment:
I think this is not true - it is not the same as before. You are finding dir for all actions. But in Nette\UI\Presenter there is this code:
```
$dir = dirname(static::getReflection()->getFileName());
$dir = is_dir("$dir/templates") ? $dir : dirname($dir);
$list = [
	"$dir/templates/$presenter/@$layout.latte",
	"$dir/templates/$presenter.@$layout.latte",
];
```

So if `$dir/templates` it is used for all actions